data/reach_rule_substrings.p

```diff
@@ -0,0 +1,364 @@
+p89
+aS'Negative_[^_]*_syntax_5_verb'
+p90
+aS'Negative_[^_]*_syntax_6_verb'
+p91
+aS'Negative_[^_]*_syntax_7_verb'
+p92
+aS'Negative_[^_]*_syntax_1_noun'
+p93
+aS'Negative_[^_]*_syntax_2_noun'
+p94
+aS'Negative_[^_]*_syntax_3_noun'
+p95
+aS'Negative_[^_]*_syntax_5_noun'
+p96
+aS'Negative_[^_]*_syntax_6_noun'
+p97
+aS'Negative_[^_]*_syntax_7_noun'
+p98
+aS'Negative_[^_]*_syntax_noun_Hearst'
+p99
+aS'Negative_[^_]*_token_1_noun'
+p100
+aS'Negative_[^_]*_token_2_noun'
+p101
+aS'Negative_[^_]*_copula_1'
+p102
+aS'Negative_[^_]*_possessive_noun'
+p103
+aS'Negative_[^_]*_token_experimental_results_1'
+p104
+aS'Positive_[^_]*_syntax_1_verb'
+p105
+aS'Positive_[^_]*_syntax_1b_verb'
+p106
+aS'Positive_[^_]*_syntax_1c_verb'
+p107
+aS'Positive_[^_]*_syntax_2_verb'
+p108
+aS'Positive_[^_]*_syntax_3_verb'
+p109
+aS'Positive_[^_]*_syntax_4_verb'
+p110
+aS'Positive_[^_]*_syntax_5_verb'
+p111
+aS'Positive_[^_]*_syntax_6_verb'
+p112
+aS'Positive_[^_]*_syntax_7_verb'
+p113
+aS'Positive_[^_]*_syntax_8_verb'
+p114
+aS'Positive_[^_]*_syntax_1_noun'
+p115
+aS'Positive_[^_]*_syntax_2_noun'
+p116
+aS'Positive_[^_]*_syntax_3_noun'
+p117
+aS'Positive_[^_]*_syntax_5_noun'
+p118
+aS'Positive_[^_]*_syntax_6_noun'
+p119
+aS'Positive_[^_]*_syntax_7_noun'
+p120
+aS'Positive_[^_]*_syntax_noun_Hearst'
+p121
+aS'Positive_[^_]*_copula_1'
+p122
+aS'Positive_[^_]*_possessive_noun'
+p123
+aS'Positive_[^_]*_token_1_verb'
+p124
+aS'Positive_[^_]*_token_1b_verb'
+p125
+aS'Positive_[^_]*_token_1_noun'
+p126
+aS'Positive_[^_]*_token_2_noun'
+p127
+aS'Positive_[^_]*_token_experimental_results_1'
+p128
+aS'Positive_[^_]*_nested_syntax_2_verb'
+p129
+aS'Positive_early_[^_]*'
+p130
+aS'binding1a'
+p131
+aS'binding1b'
+p132
+aS'binding1forms'
+p133
+aS'binding2'
+p134
+aS'binding3'
+p135
+aS'binding_of_and'
+p136
+aS'binding5'
+p137
+aS'binding6'
+p138
+aS'binding7a'
+p139
+aS'binding7b'
+p140
+aS'binding7c'
+p141
+aS'binding9'
+p142
+aS'binding10'
+p143
+aS'binding10b'
+p144
+aS'binding11'
+p145
+aS'binding12'
+p146
+aS'binding13'
+p147
+aS'binding14'
+p148
+aS'binding_16'
+p149
+aS'binding_17'
+p150
+aS'binding_18'
+p151
+aS'binding_19'
+p152
+aS'binding_20'
+p153
+aS'binding_21'
+p154
+aS'binding_syntax_23_verb'
+p155
+aS'binding_syntax_24_noun'
+p156
+aS'binding_syntax_25_verb'
+p157
+aS'binding_syntax_26_verb'
+p158
+aS'binding_syntax_27_verb'
+p159
+aS'binding_token_1'
+p160
+aS'binding_token_3'
+p161
+aS'binding_token_4'
+p162
+aS'binding_token_5'
+p163
+aS'binding_coexist'
+p164
+aS'binding_oncebound'
+p165
+aS'binding_thatisbound'
+p166
+aS'binding_thatisbound_surface'
+p167
+aS'binding_complex_formation_1'
+p168
+aS'binding_complex_A_B'
+p169
+aS'binding_activity'
+p170
+aS'complex_formation'
+p171
+aS'hydrolysis_1'
+p172
+aS'hydrolysis_2'
+p173
+aS'hydrolysis_2wrong'
+p174
+aS'hydrolysis_3'
+p175
+aS'hydrolysis_4'
+p176
+aS'hydrolysis_token_1'
+p177
+aS'hydrolysis_token_2'
+p178
+aS'hydrolysis_decl1'
+p179
+aS'hydrolysis_pass1'
+p180
+aS'hydrolysis_subjnom1'
+p181
+a.
```

Comment:
I think this file should live in indra/sources/reach, and could just be a raw .txt file since it just contains a list of strings.